### PR TITLE
Added tlg1799.tlg016

### PR DESCRIPTION
**Pull request by planatheisa**

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +53 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +10523 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1799/tlg016/__cts__.xml`                   | New      |
| `./data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml` | New      |
| `./data/tlg1799/tlg016/__cts__.xml`                   | Passing  |
| `./data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3493)*

File: data/tlg1799/tlg016/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1799" xml:lang="grc" urn="urn:cts:greekLit:tlg1799.tlg016">
+    <ti:title xml:lang="lat">Fragmenta</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg1799.tlg016.1st1K-grc1" workUrn="urn:cts:greekLit:tlg1799.tlg016">
+      <ti:label xml:lang="lat">v</ti:label>
```
Comment (sonofmun) [inline data/tlg1799/tlg016/__cts__.xml]: I assume this is not the correct title.

File: data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,1336 @@
...
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg1799.tlg.1st1K-grc1.xml</idno>
```
Comment (sonofmun) [inline data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml]: Filename is incomplete.

File: data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,1336 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg1799.tlg016.1st1K-grc1">
+                <pb n="227"/>
+                <div type="textpart" subtype="chapter" n="1">
```
Comment (sonofmun) [inline data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml]: There are several chapters here that have no sections under them. I think you should add a single "section"-level div under each of these chapters to keep the citation scheme consistent.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +64 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +10523 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg1799/tlg016/__cts__.xml`                   | New      |
| `./data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml` | New      |
| `./data/tlg1799/tlg016/__cts__.xml`                   | Passing  |
| `./data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3615)*

**Pull request merged**